int16 id
float64 coord_x
float64 coord_y
